Monika After Story/game/script-topics.rpy

```diff
@@ -3641,13 +3641,14 @@ label monika_trolley:
     m 1eua "Oh, cool. I love thinking about these sorts of thought experiments."
     m 1euc "I guess we're talking about real people, right? I wouldn't have a particular preference if I knew they didn't have free will."
     m 1dsc "Hmmm..."
-    m 1eua "If we were talking about the classic problem, of course I would switch."
-    m 1rkc "But if we're talking about real people..."
-    m 3esc "I wouldn't switch."
-    m 4esc "That's the only logical choice, if you think about it."
-    m 1euc "It feels nice to say that you'd save the most people, doesn't it? But people only say that because nobody in the example matters to them."
-    m "If they actually cared about the person, they'd kill the other five in a heartbeat."
-    m 2tsb "Do you know anyone who would want a self-driving car to save other people by killing the driver? Didn't think so, [player]."
+    m 3esa "The classic trolley problem has us choose between letting the trolley run over 5 people, or switching to the track where only 1 person will be killed."
+    m 1esb "I think this problem is too easy. Of course I would switch."
+    m 1esc "It takes a rare kind of moral cowardice not to save 5 lives, just to avoid personal responsibility for that one person's death."
```

Comment:
If you ask me, I'd prefer not to call out and attack those people who wouldn't switch. Of course, not everyone is going to agree on this question, and I don't think we should attack them.

Reply:
Multiple things come to mind here, I'll try to sum it up as best as I can.
> I agree in that the wording here might be a bit too strong.
> As you said, this is a pretty polarizing topic. But I feel like that's more of a reason to keep something like this rather than not. When people actually sit down and think about these kinds of thought experiments, they usually feel rather strongly about the answer they arrive at. Monika would presumably be the same. She is her own character with her own opinions, not a safe space.
> I feel like the number of people who'd get upset at this is a good deal less than what one might think at first glance. The community here isn't the general populace, it takes a specific kind of person play this mod in the first place.
> How about adding a milder version behind a sensitive mode check? I'd say that gets us the best of both worlds.

Reply:
Whatever conclusion you come to with the question, you're probably going to feel extremely strongly about it, and I think that's fine. The easy answer most people would go with would be to pull the lever. I would just imagine that she'd at least be respectful to the other side rather than calling out their rare moral cowardice. 

Think back to the `monika_debate` topic. Having been in debate club, and knowing that a common pitfall is seeing their own side as superior, and ending up negatively criticizing the other side, making them feel attacked and accused of things like poor taste, I think she should keep this in mind, even in polarizing questions like this. 

As someone who wouldn't pull the lever, yes, I'll admit I must be very rare, but I accept and respect that so many other people would pull the lever for their own good reasons. I don't think Monika should call moral cowardice to those who don't agree with her.

Reply:
One thing is having your own opinion on a problem, another thing is judging other people's opinions. Especially when you don't know why they have such opinion.
I don't think Monika would call the player like that only because of their choice. And she unintentionally (she doesn't know the player's opinion) might do that. It's not about will it upset one person or 10. It's a matter of why Monika wouldn't do so. IMO it's better to reword it.

Reply:
Can I suggest possibly flipping this a little? Instead of having it come off as sounding like a personal attack to those who disagree, she can be just as strong, but word it as solely her opinion. For example: ` 'There is no way I could personally justify to myself saving one person at the expense of five others, just so I wouldn't have to accept responsibility for that one person's death.'`